docs/src/templates/includes/components/core/je-previousOrderCard.hbs

```diff
@@ -0,0 +1,25 @@
+<a href="{{ restaurantLink }}"
+    class="c-previousOrderCard {{ optionalClassNames }}"
+    href="{{ restaurantLink }}"
+    title="{{ orderAgainLinkTitle }}"
+    data-gtm="home|click-previous|reorder"
+>
+
+    <div class="c-previousOrderCard-image">
+        <img src="{{ restaurantImageUrl }}" alt="" class="c-previousOrderCard-img">
+    </div>
+
+    <div class="c-previousOrderCard-content">
+        <h3 class="c-previousOrderCard-title">{{ restaurantName }}</h3>
+        <p class="c-previousOrderCard-date">{{ orderDate }}</p>
+        <div class="c-previousOrderCard-summary-grid">
+            <div class="c-previousOrderCard-summary">
+                {{ orderSummary }}
+                <strong class="c-previousOrderCard-showMore" href="#">{{ showMoreText }}</strong>
```

Comment:
Can `strong` elements use `href`?

Reply:
I don't think so. I guess this is a show more element which requires some JS to make it work. In that case we should `tabindex="0"`

Reply:
@xander-marjoram @oanuna I forgot to remove `href` after changing this element from a link. In Vue implementation there will be some js for this element. For now it is only layout and styles. I've added tabindex